quic/s2n-quic-transport/src/stream/tests/stream_managers_tests.rs

```diff
@@ -863,7 +863,9 @@ fn send_streams_blocked_frame_when_blocked_by_peer() {
             manager.get_transmission_interest()
         );
 
-        let expected_next_stream_blocked_time = write_context.current_time + DEFAULT_SYNC_PERIOD;
+        let expected_transmission_backoff = 4;
```

Comment:
not that i understood it before, but curious about the magic constant. these tests are pretty difficult to reason about since there is alot going on